packages/avatar/src/react/index.js

```diff
@@ -16,25 +16,52 @@ const styles = {
   initials: ({ name }) => glamor.css(css['.psds-avatar__initials'])
```

Comment:
lines 40-41 show main checks
lines 26-31 are the new callbacks for 
lines 48-49 the new image loading event listeners